src/start_menu/main_settings/change_sound_volume.cpp

```diff
@@ -0,0 +1,22 @@
+#include "change_sound_volume.h"
+#include "../../utils/utils.h"
+#include <iostream>
+
+void ChangeSoundVolume()
+{
+	std::cout << "Your current volume is at: " << utils::g_sound_volume << std::endl;
+	std::cout << "Please enter your desired sound volume (0-1000): ";
```

Comment:
I am actually thinking of changing this into a 0-100 basis. It'll be a somewhat uglier code, but more intuitive to the end user. What do you think, @DIONYSUS-BASOUNAS ?

Reply:
yes i think it's more user-friendly to me 0-100